src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,28 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+          <UserInfo user={user} />
```

Comment:
user can be undefined, better to add some check here
```suggestion
          {user && <UserInfo user={user} />}
```